src/core/protocol-authorization.ts

```diff
@@ -84,7 +108,12 @@ export class ProtocolAuthorization {
    * Constructs a chain of ancestor messages
    * @returns the ancestor chain of messages where the first element is the root of the chain; returns empty array if no parent is specified.
    */
-  private static async constructAncestorMessageChain(tenant: string, recordsWrite: RecordsWrite, messageStore: MessageStore)
+  private static async constructAncestorMessageChain(
+    tenant: string,
+    incomingMessage: RecordsRead | RecordsWrite,
```

Comment:
Sanity: this `incomingMessage` being used? If not, it's very odd that linter doesn't pick this up...

Reply:
Oh you're right. Yea not sure why the linter didn't catch that. Will remove